src/main/resources/templates/snowflake/shared/create-snowpipe.ssp

```diff
@@ -0,0 +1,22 @@
+#import(io.phdata.streamliner.schemadefiner.model.Snowflake)
```

Comment:
create-snowpipe is used in two templates to added it in shared folder and used in others.